Bruno-kalvium.bru

```diff
@@ -0,0 +1,11 @@
+meta {
```

Comment:
Consider adding more details in the 'meta' section such as 'description' to provide more context about the test. [medium]